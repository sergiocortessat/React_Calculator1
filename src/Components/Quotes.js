import React from 'react';
import style from '../style/Quotes.module.scss';

function Quotes() {
  return (
    <div className={style.quotes} data-testid="quote">
      <h2>Unfortunately, no one can be told what the Matrix is. You have to see it for yourself</h2>
      <p>Morpheus</p>
    </div>
  );
}

export default Quotes;
