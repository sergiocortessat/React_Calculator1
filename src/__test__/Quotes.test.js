import React from 'react';
import { BrowserRouter } from 'react-router-dom';
import renderer from 'react-test-renderer';
import Quotes from '../Components/Quotes';

it('renders correctly the quotes component', () => {
  const tree = renderer
    .create(
      <>
        <BrowserRouter>
          <Quotes />
        </BrowserRouter>
        <p>Testing the rendering of Quotes component using snapshot</p>
      </>,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
