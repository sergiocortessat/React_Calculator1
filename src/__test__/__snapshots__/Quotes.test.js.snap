// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly the quotes component 1`] = `
Array [
  <div
    className="quotes"
    data-testid="quote"
  >
    <h2>
      Unfortunately, no one can be told what the Matrix is. You have to see it for yourself
    </h2>
    <p>
      Morpheus
    </p>
  </div>,
  <p>
    Testing the rendering of Quotes component using snapshot
  </p>,
]
`;
